app/addons/documents/base.js

```diff
@@ -109,6 +109,10 @@ FauxtonAPI.registerUrls('view', {
 
   fragment: function (database, designDoc, viewName) {
     return 'database/' + database + designDoc + '/_view/' + viewName;
+  },
+
+  query: function (database, designDoc, viewName, query) {
+    return 'database/' + database + '/_design/' + designDoc + '/_view/' + viewName + getQueryParam(query);
```

Comment:
Is the designDoc and view name encoded elsewhere? will it work with `/_designDoc/view/name`?